src/services/lineService.ts

```diff
@@ -22,6 +23,7 @@ export default class LineService {
         }
     }
 
+    // TODO: rename as fetchLine
     public static async getLine(lintunnus: string): Promise<ILine | null> {
```

Comment:
Why TODO for this? Why not just change this already now, getLine and getAllLines are just called at one place respectively. So, there would not be too much work to just replace them.